js/menus/menuTemplate.js

```diff
@@ -184,17 +186,35 @@ function getTemplate(app) {
             checked: launchOnStartup,
             click: function (item) {
                 if (item.checked){
-                    symphonyAutoLauncher.enable()
-                    .catch(function (err) {
-                        let title = 'Error setting AutoLaunch configuration';
-                        electron.dialog.showErrorBox(title, title + ': ' + err);
-                    });
+                    if (isMac){
+                        childProcess.exec(`launchctl load ${launchAgentPath}`, (err) => {
```

Comment:
ok, then please add documentation here why we are using different soln (i.e., bug in electron and provide ref to bug - so we can track when it gets fixed).